help/link-to-profile.md

```diff
@@ -0,0 +1,37 @@
+# Link to a profile
+
+Zulip allows you to easily copy a direct link to someone's profile.
+
+## Link to a profile
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!right-sidebar-view-profile.md!}
+
+1. At the top of the user profile, Click on the **link** (<i class="zulip-icon zulip-icon-link"></i>)
+   located to the right of their name to copy the **user profile link**.
+
+!!! Tip ""
+
+    You can also click on a user's profile picture or name on a
+    message they sent to open their **user card**, and skip to
+    the 3rd step.
+
+{tab|mobile}
+
+1. Tap on a user's profile picture or name.
+
+2. Click **View profile**.
```

Comment:
It doesn't super matter,  but our preferred style is to number with `1.` throughout.

Reply:
Actually, wait a minute, why are there mobile instructions at all? We don't have this feature in the mobile apps.

Reply:
My sincere apologies for the confusion. I mistakenly interpreted the mobile section doc, thinking it was for the mobile version of the website instead of the app. I have now removed the mobile instructions.